### PR TITLE
add video to homepage (everything is done besides puting the correct video link)

### PR DESCRIPTION
**Pull request by kakax114**

### CODE REVIEW

File: content/_index.en.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (matt-auckland) [inline content/_index.en.html]: Why are we adding jquery in here? It's already imported in the line directly above it. 

```suggestion
```

File: content/_index.en.html [hunk 2]
```diff
@@ -20,7 +24,18 @@ <h1>Simplifying Crypto</h1>
   <section class="welcome" id="welcome">
     <h1>Welcome To NavCoin</h1>
     <div class="grid-container ninety-vw-container">
-      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <!-- <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img"> -->
```
Comment (matt-auckland) [inline content/_index.en.html]:
```suggestion
```
If we aren't using this image remove the commented code

File: content/_index.en.html [hunk 3]
```diff
@@ -20,7 +24,18 @@ <h1>Simplifying Crypto</h1>
...
+      <div class="container-eco">
+        <a class="video-eco" title="Test title" href="https://www.youtube.com/watch?v=kIC-AflIY2Q">
```
Comment (matt-auckland) [inline content/_index.en.html]: If we have the final video ready upload it to youtube as a private video and put the url in here.
WHen we are ready to release the video we can merge this PR and set the video to public on youtube.

We also need to fix the `title="Test title"`

File: content/_index.en.html [hunk 4]
```diff
@@ -222,4 +237,77 @@ <h3>Medium</h3>
...
+    }
+  }
+});
+
+
 </script>
+
+
+<style>
+.container-eco {
```
Comment (matt-auckland) [inline content/_index.en.html]: Move this CSS into the appropriate scss file please, into the homepage's scss file or `nav.scss` if we don't have a file for the homepage.

Comment (kakax114) [reply content/_index.en.html]: done

File: content/_index.en.html [hunk 5]
```diff
@@ -222,4 +235,77 @@ <h3>Medium</h3>
...
```
Comment (matt-auckland) [inline content/_index.en.html]: You added the styles to the .scss but you forgot to delete it from here 😛

**Pull request merged**